packages/angular_devkit/schematics/tasks/tslint-fix/executor.ts

```diff
@@ -79,7 +79,7 @@ function _listAllFiles(root: string): string[] {
   return result;
 }
 
-
+/** @deprecated Use `ng lint --fix` directly instead. */
```

Comment:
@kyliau can you check if this is used in Google?

Reply:
negative. Google doesn't use this task executor to fix code.

Reply:
Copy that.